data/assets/scene/solarsystem/planets/mercury/layers/colorlayers/messenger_hiw_utah.asset

```diff
@@ -6,8 +7,16 @@ local layer = {
   FilePath = asset.localResource("messenger_hiw_utah.wms")
 }
 
-asset.onInitialize(function () 
-  openspace.globebrowsing.addLayer(globeIdentifier, "ColorLayers", layer)
-end)
+assetHelper.registerLayerAndExport(asset, layer, globeIdentifier, "ColorLayers")
+
+
+asset.meta = {
+  Name = "Messenger HIW [Utah]",
+  Version = "1.0",
+  Description = [[ Mercury HIW layer for Mercury globe. 
```

Comment:
same here, "**Messenger** HIW layer for Mercury globe"

Reply:
I guess it should be MESSENGER in that case?